docs/guide/ecosystem/client-server-communication.md

```diff
@@ -43,21 +47,21 @@ To use it with Vue, you dont need plugins, you can just import `fetch` and start
 ::: tip
  This part of the page needs a bit of love. If you can, please contribute by adding the missing info.
 :::
-- what is GraphQL
+- What is GraphQL
 - When does it make sense to use it
-- links to tutorials
+- Links to tutorials
 
 ### Vue Apollo
 
 - What is Apollo
```

Comment:
This is meant for someone to add content to hehe. I am looking for volunteers :)